src/fetch.py

```
@@ -0,0 +1,43 @@
+from meteostat import Point, Daily
+from datetime import datetime
+
+from typing import Union, Optional
```

Ruff (0.11.9)

`typing.Union` imported but unused

Remove unused import: `typing.Union`

(F401)